python/Ganga/test/GPI/TutorialTests.py

```diff
@@ -106,6 +106,7 @@ def test_c_JobManipulation(self):
         # -- JOBMANIPULATION JOBSLICING STOP
 
         # -- JOBMANIPULATION RESUBMIT START
+        jobs(0).kill()
```

Comment:
This should be before the `# -- JOBMANIPULATION RESUBMIT START` line so that it doesn't show up in that doc section.